CONTRIBUTING.md

```diff
@@ -35,3 +35,44 @@ During development two SQLite databases are used:
 
 Both apps will allows you to automatically migrate the database and add seeded development data.
 Please visit both the apps in your browser, I recommend you start with the API app first.
+
+### Migrations
```

Comment:
Maybe make this `####` so it goes under `### Database`? Since it seems related. The other headings would have to be bumped down one level too.